README.rst

```diff
@@ -146,7 +162,7 @@ pre-commit
     ---
     repos:
       - repo: https://github.com/abravalheri/validate-pyproject
-        rev: main
+        rev: <insert current version here>
         hooks:
           - id: validate-pyproject
```

Comment:
```suggestion
          - id: validate-pyproject
          # Uncomment if you want to use extra validations from SchemaStore:
          # additional_dependencies: ["validate-pyproject-schema-store[all]"]
```

Reply:
Needs to have the "all" extra, since putting anything in this entry overrides the `[all]` we provide.